plugins/modules/zos_backup_restore.py

```diff
@@ -456,6 +457,7 @@ def main():
                 sms_management_class=sms_management_class,
                 tmp_hlq=tmp_hlq,
             )
+            result["backup_name"] = backup_name
```

Comment:
There is no need to put these inside each of right? We can just put them outside the if statement and cover both cases